src/goto-instrument/reachability_slicer.h

```diff
@@ -33,13 +37,20 @@ void reachability_slicer(
   const bool include_forward_reachability);
 
 // clang-format off
-#define OPT_REACHABILITY_SLICER \
-  "(reachability-slice)(reachability-slice-fb)" // NOLINT(*)
-
-#define HELP_REACHABILITY_SLICER \
-  " --reachability-slice         remove instructions that cannot appear on\n" \
-  "                              a trace from entry point to a property\n" \
-  " --reachability-slice-fb      remove instructions that cannot appear on\n" \
-  "                              a trace from entry point through a property\n" // NOLINT(*)
+#define OPT_REACHABILITY_SLICER                                                \
+  "(fp-reachability-slice):(reachability-slice)(reachability-slice-fb)" // NOLINT(*)
+
+#define HELP_REACHABILITY_SLICER                                               \
+  " --fp-reachability-slice f    remove instructions that cannot appear on a " \
+  "trace\n                              that visits all given functions. The " \
```

Comment:
The layout looks a bit odd. Since clang-format is already off for this section, I'm sure you can make this nicer manually.